spec/lib/tasks/scripts/telemedicine_reports_spec.rb

```diff
@@ -11,6 +11,7 @@
   let!(:user_2) { create(:user, id: "31f4b6c4-6172-499a-95e1-bbbbbbbbbbbb", registration_facility: facility_2) }
   let!(:high_bps) { create_list(:blood_pressure, 2, :hypertensive, :with_encounter, facility: facility_1, user: user_1, recorded_at: period_start + 1.day) }
   let!(:high_bps_2) { create_list(:blood_pressure, 2, :with_encounter, :hypertensive, facility: facility_2, user: user_2, recorded_at: period_start + 1.day) }
+  let!(:teleconsultation_request) { create(:teleconsultation, facility: facility_1, patient: high_bps.first.patient, requester: user_1, requested_medical_officer: user_2, medical_officer: user_2, device_created_at: Date.parse("2020-08-04").beginning_of_day) }
```

Comment:
Could we make another couple TC requests? One with `nil` record info, one with "requester" `completed: true`?

Reply:
I'm going off of [this request](https://simpledotorg.slack.com/archives/CV9V1DWLV/p1602604499062900?thread_ts=1602600744.062700&cid=CV9V1DWLV) from Dr Reena atm, we can add more columns iteratively if they become valuable.

Reply:
Oh, sorry, that's not what I meant. I meant, for this test case, can we make more TC request records with different combinations of data, so we can ensure we're putting the right number into the column?

Reply:
Ah, took me a while, even after reading this clarification 😅. I'll do this.

Reply:
Man, I'm terrible at clarification 🤦 Thanks!